x-pack/legacy/plugins/ml/common/util/parse_interval.ts

```diff
@@ -4,17 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-
-
-import moment from 'moment';
+import { duration, Duration, unitOfTime } from 'moment';
 import dateMath from '@elastic/datemath';
 
+type SupportedUnits = unitOfTime.Base;
```

Comment:
Just wondering if this should be singular `SupportedUnit` since later on it's used as an array like `SupportedUnits[]`.

Reply:
`unitOfTime.Base` is a list of string literal types, something like `ms | s | m | h |....`
so from a language point of view i thought the plural makes sense.
but i guess it's a union so it'll only have one value.